apps/portal/src/components/common/ProductsSection.js

```diff
@@ -847,9 +858,19 @@ function YearlyDiscount({discount}) {
     }
 }
 
-function ProductPriceSwitch({selectedInterval, setSelectedInterval}) {
+function ProductPriceSwitch({selectedInterval, setSelectedInterval, products}) {
     const {site, t} = useContext(AppContext);
     const {portal_plans: portalPlans} = site;
+
+    // Filter only paid products
+    const paidProducts = products.filter(product => product.id !== 'free');
```

Comment:
@SimonBackx I tried this, but it complained that `amount` is undefined in case of free product, so I’ve added `?` like this `product.monthlyPrice?.amount` to check if it’s undefined. Makes sense? 🙏

Reply:
I would just keep the filter, but change it to what I suggested